server/views/pages/strengths/select-category/index.njk

```diff
@@ -1,7 +1,7 @@
 {% extends "../layout.njk" %}
 
-{% set pageId = 'create-strenth-select-category' %}
-{% set pageTitle = "Add a strenth - select category" %}
```

Comment:
ha - I also have a 'need' when it comes to the word strength - I almost always write it as strenght and takes ages to see why it's wrong.